src/app/market/sell/add-item/add-item.component.html

```diff
@@ -150,11 +150,11 @@
           <input matInput placeholder="Quantity">
         </mat-form-field-->
         <mat-form-field fxFlex="50%" class="auto-width">
-          <input matInput min="0" placeholder="Domestic shipping price" formControlName="domesticShippingPrice" type="number">
+          <input matInput min="0" (keypress)="inputValidate($event)" placeholder="Domestic shipping price *" formControlName="domesticShippingPrice" type="number">
```

Comment:
This still leaves the option of copy/pasting values into the input. This can be mitigated by including a (paste) event handler (possibly pointing to the same method - this would require updating the validation method to include a check for the pasted data  (see comment with the example on the function itself).